live/tests/acceptance/a1-page-accueil-test.js

```diff
@@ -1,31 +1,79 @@
-import { describe, it, beforeEach, afterEach } from 'mocha';
+import { describe, it, before, after } from 'mocha';
```

Comment:
Bien vu. Je pensais avoir supprimé les tests inutiles. 

Ce que je ne sais pas tester en intégration : 

- la navigation entre pages
- la génération de liens quand le template comporte des `{{linnk-to}}` car les initializers (notamment celui qui transforme les link-to via le router) ne sont pas injectés (et son difficilement injectables) dans les tests d'intégration / unitaires.